### ISSUE TITLE
bwa mem flags could be problematic

### ISSUE THREAD

**Issue by d-cameron** [Mangul-Lab-USC/benchmarking_SV_publication#7]
`-a` can have a significant impact on caller performance. In the case of GRIDSS, results were signficantly worse when `-a` was an option.

`-Y` is similarly problematic: some older callers pre-date the supplementary flag being added to the SAM specifications so essentially require `-Y` to treat them fairly. Recent callers may actually adhere to the SAM specifications so `-Y` needs to not be supplied to those callers.

Other callers can rely on specific aligner versions. When I last benchmarked, I found that LUMPY sensitivity got halved due to a minor version change in version of bwa used for alignment.

In summary, `-a` is a problematic default, and `-Y` may or may not be required for older callers.

**Comment by Addicted-to-coding** [comment]
Thanks a lot for your feedback, we have run bwa using the -M option. We have updated our manuscript with this detail.
